tests/db_functions/test_nullif.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import Value
+from django.db.models.functions import NullIf
+from django.test import TestCase
+
+from .models import Author
+
+
+class NullIfTests(TestCase):
+    def setUp(self):
+        Author.objects.create(name='John Smith', alias='smithj')
+        Author.objects.create(name='Rhonda', alias='Rhonda')
+
+    def test_nullif(self):
+        authors = Author.objects.annotate(nullif=NullIf('alias', 'name')).values_list('nullif')
+
+        self.assertSequenceEqual(
+            authors, [
+                ('smithj',),
+                (None,),
```

Comment:
On Oracle, this returns an empty string instead of None.

Reply:
OK. That will essentially require two separate tests, one for Oracle, and one for the other backends?